modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -133,6 +134,47 @@ public void testAddObjectRelationship() throws Exception {
 		Assert.assertFalse(_hasTable(objectRelationship.getDBTableName()));
 	}
 
+	@Test
+	public void testAddObjectRelationshipMappingTableValues() throws Exception {
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				StringUtil.randomId(),
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		_objectRelationshipLocalService.addObjectRelationshipMappingTableValues(
+			objectRelationship.getUserId(),
+			objectRelationship.getObjectRelationshipId(),
+			_objectDefinition1.getPrimaryKey(),
+			_objectDefinition1.getPrimaryKey(), null);
+
+		List<ObjectRelationship> objectRelationshipList =
```

Comment:
Could you rename this var to "objectRelationships", please?
Sorry for the delay in reviewing this part! :/